### PR TITLE
V4 release

### PR DESCRIPTION
**Pull request by twiecki**
Getting everything ready for the release.

### CODE REVIEW

File: blog/v4_announcement.ipynb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
```
Comment (aloctavodia) [inline blog/v4_announcement.ipynb:1]: <p>Mentioning PyMC4 could add to the confusion, could we just forget about it? :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (aloctavodia) [inline blog/v4_announcement.ipynb:1]: <p>Could we get rid of this warnings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (aloctavodia) [inline blog/v4_announcement.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    import arviz as az
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (aloctavodia) [inline blog/v4_announcement.ipynb:1]: <p>we may add the meme here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (aloctavodia) [inline blog/v4_announcement.ipynb:1]: <p>Move import to the first cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (aloctavodia) [inline blog/v4_announcement.ipynb:1]: <p>Maybe add <code>aehmc</code>, <code>aemcmc</code> ?</p><p> </p><p>Also we may want to add here, that we now have a pymc-experimental for  newly-implemented statistical methods and code not yet useful in "production"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (OriolAbril) [reply blog/v4_announcement.ipynb:1]: +1 on mentioning pymc-experimental! We already had a couple questions on discourse about BART which is now there for example

**Pull request closed without merging**